app/src/main/java/com/simplecity/amp_library/ShuttleApplication.java

```diff
@@ -93,11 +94,8 @@ public void onCreate() {
                 .create(this)
                 .inject(this);
 
-        if (LeakCanary.isInAnalyzerProcess(this)) {
-            // This process is dedicated to LeakCanary for heap analysis.
-            // You should not init your app in this process.
-            return;
-        }
+        ((ShuttleApplication) getApplicationContext()).setIsUpgraded(true);
```

Comment:
Why is the LeakCanary code removed here?

I think a simpler/safer solution would be to do:

`private boolean isUpgraded = true;`

...in the class body, and then make `setIsUpgraded()` a no-op (you can just comment out the code there).

Reply:
Ah! I might have highlighted the block by mistake. 
It definitely seems safer. Will do that.